model/flow/seal.go

```diff
@@ -14,13 +14,15 @@ type Seal struct {
 // encodable with RLP
 func (s Seal) Body() interface{} {
 	return struct {
-		BlockID    Identifier
-		ResultID   Identifier
-		FinalState StateCommitment
+		BlockID                Identifier
+		ResultID               Identifier
+		FinalState             StateCommitment
+		AggregatedApprovalSigs []AggregatedSignature
```

Comment:
Would be better if we could explain why here, like: 
```suggestion
		// otherwise, a wrong seal on a different fork for the same result with different approver assignment might be accepted.
		AggregatedApprovalSigs []AggregatedSignature
```

Reply:
Included a detailed doc for the seal